mod_form.php

```diff
@@ -69,6 +69,7 @@ public function definition() {
         $context = context_course::instance($course->id);
         // UI configuration options.
         $cfg = \mod_bigbluebuttonbn\locallib\config::get_options();
+error_log("RABSER DEBUG ".print_r($cfg,true));
```

Comment:
Debugging code has to be removed. This is going to clean a few errors that are making travis to fail.